tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -67,10 +95,12 @@ class TransformListener
     const rclcpp::QoS & qos = DynamicListenerQoS(),
     const rclcpp::QoS & static_qos = StaticListenerQoS(),
     const rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> & options =
-    rclcpp::SubscriptionOptionsWithAllocator<AllocatorT>())
+    detail::get_default_transform_listener_sub_options<AllocatorT>(),
+    const rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> & static_options =
+    detail::get_default_transform_listener_static_sub_options<AllocatorT>())
```

Comment:
Oh, I see.  You want to be able to override these options from the caller that constructs `TransformListener`.  That makes sense.